### ISSUE TITLE
Problem with importing and using datasets

### ISSUE THREAD

**Issue by JoeJoe1313** [sdv-dev/Copulas#255]
### Environment Details

Please indicate the following details about the environment in which you found the bug:

* Copulas version: 0.5.0
* Python version: 3.8.5
* Operating System: macOS Big Sur 11.4 (20F71)

### Error Description

I am trying to import copulas datasets. A few weeks ago everything worked fine. Now I am not able to do anything different from `import copulas`. I made new virtual environments, tried with different python versions and nothing is happening.

```
from copulas.datasets import sample_trivariate_xyz

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-15-64a62be211b4> in <module>
----> 1 from copulas.datasets import sample_trivariate_xyz

ModuleNotFoundError: No module named 'copulas.datasets'; 'copulas' is not a package
```

**Comment by fealho** [comment]
Hi @JoeJoe1313. A bit of a long shot, but do you happen to have a file called `copulas.py` in your working directory? If so, delete/rename it and that will fix the issue. Otherwise, let's try the following:

1. Create a new virtual environment.
2. Install copulas via pip (or conda).
3. Run the following commands and paste the result here:

```
import copulas
print(copulas.__version__)
print(copulas.__file__)
```

This will help me figure out what the issue is.

**Comment by JoeJoe1313** [comment]
Yes, I haven't realised I actually do have a ```copulas.py``` file and that it can mess up with the package. Thank you!
